adage/adage/config.py.template

```diff
@@ -136,9 +141,30 @@ CI_CONFIG.update({
                         'PORT': '5434',  # v. 9.4
                      },
     'data':          {
-                        'data_dir': DATA_CONFIG['data_dir'],
-                        'annotation_file': DATA_CONFIG['annotation_file'],
+                        'data_dir': '/home/rof/clone/data/',
+                        'annotation_file':
+                            '/home/rof/clone/data/PseudomonasAnnotation.tsv',
                      },
 })
 
+CIRCLECI_CONFIG = TEST_CONFIG.copy()
+CIRCLECI_CONFIG.update({
+    'django_key':    str(random.randint(0, 1000000)),
+    'databases':     {'default': {
+                        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+                        # database name and user cannot have upper case letters
+                        'NAME': 'circleci_test',
+                        'USER': 'ubuntu',
+                        'PASSWORD': '',
+                        'HOST': '127.0.0.1',
+                        'PORT': '5434'
+                        }
+                     },
+    'data':          {
+                        'data_dir': os.path.join(REPO_ROOT, 'data'),
```

Comment:
Oh - I see - this got added here. What do you think about a (minor) reduction in code duplication by using a single CI section and then just updating the 'databases' key?